### DISCUSSION TITLE
Nicehash Asicboost Not working

### DISCUSSION THREAD

**Discussion by elsanosi** [coinfoundry/miningcore#1438] [Q&A]
When connecting sha256Asicboost from nicehash received Error: Unknown Message ..
Something is wrong, sha256d work as normal, any help please?
Latest Release tested

**Comment by elsanosi** [comment]
@oliverw ?

**Comment by elsanosi** [comment]
[bch1] [0HML5BNP6SUEU] Accepting connection from 45.133.147.215:46698 ...                                               │
│[2022-10-03 18:40:14.4286] [I] [bch1] [0HML5BNP6SUEU] Connection from 45.133.147.215:46698 accepted on port 9905                                       │
│[2022-10-03 18:40:14.4286] [D] [bch1] [0HML5BNP6SUEU] [NET] Waiting for data ...                                                                       │
│[2022-10-03 18:40:14.4286] [D] [bch1] [0HML5BNP6SUEU] [NET] Received data: {"id":0,"method":"mining.configure","params":[["version-rolling"],{"version-│
│                                                                                                                                                       │
│[2022-10-03 18:40:14.4286] [D] [bch1] [0HML5BNP6SUEU] [NET] Waiting for data ...                                                                       │
│[2022-10-03 18:40:14.4344] [D] [bch1] [0HML5BNP6SUEU] [PIPE] Waiting for data ...                                                                      │
│[2022-10-03 18:40:14.4344] [D] [bch1] [0HML5BNP6SUEU] [PIPE] Received data: {"id":0,"method":"mining.configure","params":[["version-rolling"],{"version│
│                                                                                                                                                       │
│[2022-10-03 18:40:14.4410] [D] [bch1] [0HML5BNP6SUEU] Dispatching request 'mining.configure' [0]                                                       │
│[2022-10-03 18:40:14.4410] [I] [bch1] [0HML5BNP6SUEU] Using version-rolling mask 1fffe000                                                              │
│[2022-10-03 18:40:14.4476] [D] [bch1] [0HML5BNP6SUEU] [PIPE] Waiting for data ...                                                                      │
│[2022-10-03 18:40:14.4558] [D] [bch1] [0HML5BNP6SUEU] Sending: {"result":{"version-rolling":true,"version-rolling.mask":"1fffe000"},"id":0}            │
│[2022-10-03 18:40:26.8738] [D] [bch1] Vardiff Idle Update pass begins                                                                                  │
│[2022-10-03 18:40:26.8738] [D] [bch1] Vardiff Idle Update pass ends                                                                                    │
│[2022-10-03 18:40:44.4221] [D] [bch1] [0HML5BNP6SUEU] Received EOF                                                                                     │
│[2022-10-03 18:40:44.4275] [I] [bch1] [0HML5BNP6SUEU] Connection closed

**Reply by elsanosi** [reply]
NICEHASH ERROR:
++++++++++++++++++++++++++++++++
Pool verification process is complete. Tested pool is incompatible!

Pool host
xxxxxxx
Pool port
xxxx
Pool user
xxxxxxxx
Pool pass
X
Algorithm
SHA256AsicBoost
Selected mining algorithm: SHA256AsicBoost... OK
Sending mining subscribe request... OK
Sending login information... OK
Unknown message... Error
CLOSE

**Comment by Konstantin35** [comment]


**Reply by elsanosi** [reply]
This happened in Zcash also, Pool Error 
Any amendment to the code required?

**Comment by elsanosi** [comment]
??

**Reply by goodthebest** [reply]
@elsanosi did you manage to go through nicehash asicboost issue? Please share your solution.
